app-editors/liteide/spec

```diff
@@ -0,0 +1,4 @@
+VER=38.3
+
+SRCS="git::commit=35a0dcd::https://github.com/visualfc/liteide.git"
+CHKSUMS="SKIP"
```

Comment:
Please mark `CHKUPDATE`, see https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md and use the `anitya` type.